BitmovinPlayerSSAI/BitmovinPlayerSSAI/BitmovinPlayerSSAIApp.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  BitmovinPlayerSSAIApp.swift
+//  BitmovinPlayerSSAI
+//
+//  Created by Thomas Sablattnig on 16.07.24.
+//
```

Comment:
IMO we can remove them. Or use the Bitmovin standard file header.

Reply:
Removed: https://github.com/bitmovin/bitmovin-analytics-collector-ios-samples/pull/8/commits/7af0cad382bff5b4722ba478552621a49330e846